lib/codegen.js

```diff
@@ -53,6 +53,11 @@ var getViewForSwagger2 = function(opts, type){
             };
             op.parameters = op.parameters ? op.parameters : [];
             op.parameters.forEach(function(parameter) {
+                if ('$ref' in parameter) { 
```

Comment:
```in``` is a javascript operator that checks if a key exists in an object. I have been using for a long time, so as far as I know it if fine to use.

So this code checks if  a ```$ref``` key exists in the ```parameter``` object and returns a boolean value.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/in